data/hai/unfettered 1 alphas.txt

```diff
@@ -1098,7 +1098,7 @@ mission "<uhai> 8: It's The Alphas"
 			label "okay boy!"
 			action
 				set "unfettered: it's the Alphas!"
-			`You head to the given coordinates on the planet, but you cannot see anyone there. You wait around for an hour or so until, looking around, you finally spot it: there is a graffiti on the wall, right at the given coordinate. At first you did not pay too much attention to it because this kind of thing is common on pirate worlds, but now you notice it depicts a green human. The message is clear.`
+			`The coordinates lead you to an enclave on the fringes of town. It's an outdoor beer garden and the regulars eye you with suspicion. You order a drink and grab a table. No one approaches you, so you just look around and people watch. The only thing that catches your eye is a mural painted on the wall of the building opposite. It's a take on St George fighting the dragon. But in this version, the dragon has a plasma rifle and St George has green skin.`
```

Comment:
I do like this angle on things, a lot, actually. Really good use of iconography to showcase growing religious reverence for Alohas within this pirate section.

But I'm not sure George as dragon slayer is a common enough reference today (much less in another thousand years) to be explicitly recognized without reference like this. How about describing the scene itself a bit more, and either appeal the Alpha's description to something iconic to George specifically (though I don't know it well enough to say what that might be), or have it simply include a title.

In expanding that, perhaps instead of it referring to it directly as a plasma cannon, it's instead "similar to a plasma cannon, but [xyz differences]", geared towards appealing to a thermal repeater or rifle?

Reply:
There is a lot of potential here indeed
Issue is if its recent it'd prob be touchy and if its old it will be forgotten, except if it is really iconic.

It does need to be clear for any humans though, even if it takes a bit of thinking